Precision error in test_feature_center_of_mass on 32-bit Linux

http://nipy.bic.berkeley.edu/builders/dipy-py2.6-32/builds/511/steps/shell_6/logs/stdio

```
======================================================================
FAIL: dipy.segment.tests.test_feature.test_feature_center_of_mass
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/buildslave/bongoslave/dipy-py2_6-32/build/venv/local/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/buildslave/bongoslave/dipy-py2_6-32/build/venv/local/lib/python2.7/site-packages/dipy/segment/tests/test_feature.py", line 68, in test_feature_center_of_mass
    assert_array_equal(features, np.mean(s, axis=0)[None, :])
  File "/usr/lib/python2.7/dist-packages/numpy/testing/utils.py", line 707, in assert_array_equal
    verbose=verbose, header='Arrays are not equal')
  File "/usr/lib/python2.7/dist-packages/numpy/testing/utils.py", line 636, in assert_array_compare
    raise AssertionError(msg)
AssertionError: 
Arrays are not equal

(mismatch 33.3333333333%)
 x: array([[ 0.46466017,  0.61721522,  0.51609766]], dtype=float32)
 y: array([[ 0.46466017,  0.61721528,  0.51609766]], dtype=float32)
```

--- a/dipy/segment/tests/test_feature.py
+++ b/dipy/segment/tests/test_feature.py
@@ -65,7 +65,7 @@ def test_feature_center_of_mass():
             # Test method extract
             features = feature.extract(s)
             assert_equal(features.shape, (1, s.shape[1]))
-            assert_array_equal(features, np.mean(s, axis=0)[None, :])
+            assert_array_almost_equal(features, np.mean(s, axis=0)[None, :])
 
         # This feature type is order invariant
         assert_true(feature.is_order_invariant)

